src/components/CardComponent.jsx

```diff
@@ -1,24 +1,24 @@
+import { useContext } from 'react'
+
+import ThemeContext from '../context/ThemeContext'
+import Card from './Card'
 const CardComponent = ({searchData}) => {
+
+  const {theme, setTheme} = useContext(ThemeContext)
+  
   if(!searchData) return null
```

Comment:
instead of you can give a no-result component. Good for user experience. user knows there is a output for his every input